### PR TITLE
Update Replicate and Trim documentation, add unixfilesystem voting documentation

### PR DESCRIPTION
**Pull request by alanking**
Addresses irods/irods#6940
In service of irods/irods#7468
In service of irods/irods#7476

Still need to implement/merge changes before all of this is true. Think of this as requirements gathering :)

### CODE REVIEW

**Comment by trel** [comment]
in first commit message.....     secion ->  section

File: docs/plugins/composable_resources.md
Comment (trel) [inline docs/plugins/composable_resources.md:333]: is this true?  i'd expect intermediate would trump the replica access token correctness...

File: docs/system_overview/data_objects.md
Comment (trel) [inline docs/system_overview/data_objects.md:292]: please capitalize start and end

File: docs/system_overview/data_objects.md
Comment (trel) [inline docs/system_overview/data_objects.md:299]: good replica is older -> good replicas is older

File: docs/system_overview/data_objects.md
Comment (trel) [inline docs/system_overview/data_objects.md:303]: good replica is older -> good replicas is older

File: docs/system_overview/data_objects.md
Comment (alanking) [reply docs/system_overview/data_objects.md]: Done

File: docs/system_overview/data_objects.md
Comment (alanking) [reply docs/system_overview/data_objects.md]: Done

File: docs/plugins/composable_resources.md
Comment (alanking) [reply docs/plugins/composable_resources.md:333]: Are you referring to the fact that this falls under the "read" operation? Currently, as implemented, this is true. I don't think the open API currently cares about replica access tokens for anything other than writes, so I'm not sure that you would ever hit this case with our existing APIs.

Perhaps for the purposes of documentation we should omit the mention of the replica access tokens for the read operation as they should not apply (even though that's how it's implemented).

Comment (alanking) [reply docs/plugins/composable_resources.md]: I removed the mention from the read and unlink operations. Unlink REALLY doesn't get a replica access token because it's not even opening the replica.

Comment (trel) [reply docs/plugins/composable_resources.md]: yeps

File: docs/system_overview/data_objects.md
Comment (trel) [inline docs/system_overview/data_objects.md:271]: remain -> retain?

Comment (alanking) [reply docs/system_overview/data_objects.md:271]: Ah, yes.

**Comment by alanking** [comment]
Alright, I think this is in a good spot and reflects how things will be in 4.3.2. Fixed commit message typo. Marking ready for review.

File: docs/system_overview/data_objects.md
Comment (trel) [inline docs/system_overview/data_objects.md:277]: i don't think the last sentence is correct for this next table... there is no string of multiple characters.

Comment (alanking) [reply docs/system_overview/data_objects.md:277]: Hmm... I see what you mean. How's this?
```suggestion
In the following cases, ‘X’ represents a stale replica, ‘&’ represents a good replica, and ‘-’ indicates a replica which does not exist. This table shows the result of a trim targeting a data object with 2 replicas and a minimum replica count of 1.
```

Comment (trel) [reply docs/system_overview/data_objects.md:277]: yep - seems to cover it.

**Review by korydraughn — APPROVED** [review]
No complaints here.

Let's wait for one more approval before squashing/pounding.

**Comment by trel** [comment]
yes, looks good - please to squash.

**Pull request merged**
